docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -83,6 +105,26 @@ General-purpose aggregation functions
     Returns ``True`` if at least one input value is true, ``None`` if all
     values are null or if there are no values, otherwise ``False``.
 
+    Example:
+    	
+	Assume ``some_boolean_field`` is a non-nullable BooleanField and
+	there exists at least one instance of ``SomeModel`` where
+	``some_boolean_field = True``:
+        
+	>>> SomeModel.objects.aggregate(boolor=BoolOr('some_bool_field')) 
+	{'boolor': True}
+
+    BoolOr can be used with :ref:`Q objects<complex-lookups-with-q>`.
+    
+    Example:
+
+	Assume ``some_int_field`` is a non-nullable IntegerField and there
+	exists at least one instance of ``SomeModel`` where ``some_int_field``
+	is a value greater than 2:
+
+	>>> SomeModel.objects.aggregate(boolor=BoolOr(Q('some_int_field__gt=2')))
+	{'boolor': True}
```

Comment:
```suggestion
    Usage example::
        
        class Comment(models.Model):
            body = models.TextField()
            published = models.BooleanField()
            rank = models.IntegerField()
        
        >>> from django.contrib.postgres.aggregates import BitOr
        >>> Comment.objects.aggregate(boolor=BoolOr('published')
        {'boolor': True}
        >>> Comment.objects.aggregate(boolor=BoolOr(Q(rank__gt=2)))
        {'boolor': False}
```

Reply:
I've prepared edits, so I can push them to you branch, if you agree with my suggestions.

Reply:
@felixxm I like those suggestions!  They are bit more concise and to the point, in my opinion.